portfolio/src/main/webapp/style.css

```
@@ -1,9 +1,117 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+html {
+  scroll-behavior: smooth;
+  overflow-x: hidden;
+  background: #0a192f;
+  font-family: 'Ubuntu', sans-serif;
+
+}
+.header{
+    width: 100%;
+    margin: 0 auto;
+    position: fixed;
+}
+header{
+    font-weight: bold;
+    font-size: medium;
+    margin: 0;
+    
+}
+nav{
+    align-self:left;
+}
+nav ul{
+    margin:0;
+    padding: 0;
+    list-style: none;
+}
+nav li{
+    display: inline-block;
+    align-self: middle;
+    margin-right: 70px;
+    margin-left: 5px;
+    padding-top: 25px;
+    padding-bottom: 25px;
+}
+nav a{
+    color: #E0E0E0;
+    text-decoration: none;
+    text-transform: uppercase;
+    
+}
+nav a:hover{
+    text-underline-position: below;
+    color: #8B0000;
+}
+h1{
+    margin: 100px;
+    text-align: center;
+    padding: 0;
 }
+.home{
+    display: flex;
+    background-position-y: center;
+    background: url(images/Personal_picture.png);
+    background-repeat: no-repeat;
+    background-size: 400px 500px;
+    background-position-x: 85%;
+    height: 100vh;
+    background-attachment: fixed;
+    font-family: 'Ubuntu', sans-serif;
+}
+.home .max-width{
+    margin: auto 0 auto 30px;
+}
+.home .home-content .text-1{
+    font-size: 27px;
+    color: #E0E0E0;
+}
+.home .home-content .text-2{
+    font-size: 75px;
+    font-weight: 600;
+    margin-left: -3px;
+    color:#8B0000;
+}
+.home .home-content .text-3{
+    font-size: 20px;
+    margin: 5px 0;
+    color: #E0E0E0;
+}
+.home .home-content .button-home
+{
+    background-color: #8B0000;
+    color: #E0E0E0;
+    text-align: center;
+    padding: 15px 32px;
+    display: inline-block;
+    margin: 4px 2px;
+    border-radius: 12px;
+    text-decoration: none;
+    border: none;
+    cursor: pointer;
+}
+#about{
+    height:700px;
+    margin-top: 20px;
+    display: inline-block;
+    color: #E0E0E0;
+    width: 100%;
+}
+.about-content
+{
```

The opening `{` should be on the same line as the selector
```suggestion
.about-content {
```